scripts/afterPack.js

```diff
@@ -2,15 +2,19 @@ const path = require('node:path');
 const fs = require('node:fs');
 const { AfterPackContext } = require('electron-builder');
 
-const packageJson = require('../package.json');
-const electronLanguages = packageJson.build.electronLanguages;
+const builderConfig = require('../config/electron-builder');
+const electronLanguages = builderConfig.electronLanguages;
+
+function logAfterPackProgress(msg) {
+  // biome-ignore lint/suspicious/noConsoleLog: log notarizing progress
+  console.log(`  • [afterSign]: ${msg}`);
```

Comment:
This previously said afterPack, and the function name matches that, but the log now says afterSign. Is that intentional?

Reply:
Ah great catch.  Copy/paste error